scrapy/core/downloader/handlers/http11.py

```diff
@@ -151,23 +157,22 @@ def connect(self, protocolFactory):
         return self._tunnelReadyDeferred
 
 
-def tunnel_request_data(host, port, proxy_auth_header=None):
+def tunnel_request_data(host, port, proxy_auth_header=None, host_header=True):
```

Comment:
Not really needed. Definitely an artifact from my tests, switching this on and off.
It's a boolean because it doesn't convey a string value, but tells to add host string or not, compared to proxy_auth_header which is a string.
I can remove it.